test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -189,14 +189,14 @@ def train_mnist_model(serving_folder_path, ec2_connection, python_invoker):
 def host_setup_for_tensorflow_inference(
     serving_folder_path, framework_version, ec2_connection, is_neuron, is_graviton, model_name, python_invoker
 ):
-    # Installing protobuf at 3.20.* to fix errors in using latest protobuf
-    ec2_connection.run((f"{python_invoker} -m pip install --user -qq -U 'protobuf>=3.20,<3.21'"), hide=True)
+    # Install PIP so we can test
+    ec2_connection.run((f"sudo apt-get update && sudo apt-get install -y python3-pip"), hide=True)
```

Comment:
Does this fix handle the installation of the right version of protobuf?

Reply:
yes as this installs pip on the testing host, allowing for protobuf to be installed on the testing host. mind, this is not the container, but the host which is testing the container.